patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -443,6 +443,11 @@
 +
 +
 +    /* ======================================== FORGE START =====================================*/
```

Comment:
do we still need the other opengui ?
https://github.com/ostenvighx/MinecraftForge/blob/126bffadc41ba5e1564f772eb2a215523756b229/patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch#L428-L442